okonomiyaki/utils/tests/test_test_data.py

```diff
@@ -0,0 +1,43 @@
+import os
+import sys
+import zipfile2
+
+from okonomiyaki.runtimes.runtime_metadata import IRuntimeMetadata
+
+from .. import test_data
+
+if sys.version_info < (2, 7):
+    import unittest2 as unittest
+else:
+    import unittest
+
+
+class TestDummyPythonRuntimes(unittest.TestCase):
+
+    def _get_contents_of_runtime(self, runtime):
+        with zipfile2.ZipFile(runtime) as zp:
+            return zp.namelist()
+
+    def test_pythonw_in_dummy_runtime(self):
+        """
+        Ensure that pythonw.exe is included in all Windows runtimes
+        """
+
+        # Given
+        python_runtimes = [attribute for attribute in dir(test_data)
```

Comment:
I am still a bit worried about relying on the variable name. If you really want to keep the "automatic discovery" magic, I would advise to just get all the attributes and filter it out where it ends with a `.runtime`:

``` python
test_data_vars = [attribute for attribute in dir(test_data) if getattr(test_data, attribute).endswith(".runtime")]
...
```